requirements/devel.txt

```diff
@@ -1,8 +1,19 @@
 # use mandatory dependencies
--r ../requirements.txt
+#-r ../requirements.txt
```

Comment:
Why are these commented out? Maybe makes sense to just delete them if not needed

Reply:
I had some collisions before, but let's it agin